src/components/Button.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react'
+import classnames, {
+  alignItems,
+  backgroundColor,
+  borderRadius,
+  boxShadow,
+  display,
+  dropShadow,
+  flexDirection,
+  fontFamily,
+  fontSize,
+  fontWeight,
+  outlineStyle,
+  padding,
+  textColor,
+  textDecoration,
+  width,
+} from 'classnames/tailwind'
+
+const commonClasses = () =>
```

Comment:
Doesn't have to be a function :) just do `const commonClasses = classnames(...)`

Reply:
Removed unused function